config/config.exs

```diff
@@ -7,8 +7,8 @@ config :logger,
   handle_sasl_reports: false
 
 config :logger, :console,
-  format: "\n$dateT$time [$level] $metadata$levelpad$message\n",
-  metadata: [:pid]
+  format: {MongoosePush.Logger, :format},
```

Comment:
I would strongly suggest to call this module based on the actual format it produces, as there may be many (and probably will).

Reply:
I thought both `fmt` and `json` will have dedicated functions inside the same module - do we want to have a module for each logger formatter?

Reply:
Yeah, event if that's not required (isn't formatter a behavior?), I would vote for them to be different modules.